telegram/ext/updater.py

```diff
@@ -112,7 +112,7 @@ def _thread_wrapper(self, target, *args, **kwargs):
     def start_polling(self,
                       poll_interval=0.0,
                       timeout=10,
-                      network_delay=2,
+                      total_timeout=15,
```

Comment:
as agreed on the dev group, we'll do `warning.warn` to inform the user the `network_delay` had been ignored.